sources/HUBViewModelDiff.m

```diff
@@ -37,6 +38,12 @@
     return [indexPaths copy];
 }
 
+@interface  HUBViewModelDiff ()
+
+@property (nonatomic, assign) BOOL headerComponentIdentifierHasChanged;
```

Comment:
I actually made the opposite comment. None of the code that uses the diff cares which part of it has actually changed, so I don't think we need to change the API of this class.